erizo_controller/erizoClient/src/Stream.js

```diff
@@ -291,6 +291,24 @@ Erizo.Stream = function (spec) {
         that.pc.updateSpec(config, callback);
     };
 
+    controlHandler = function (handlers, publisherSide, enable) {
+      publisherSide = !(publisherSide !== true);
+      var handlers = (typeof handlers === 'string') ? [handlers] : handlers;
+      handlers = (handlers instanceof Array) ? handlers : [];
+
+      if (handlers.length > 0) {
```

Comment:
We should, at some point, agree to a unified error reporting for publicly exposed API functions.